modules/search/client/components/UsersResults.js

```diff
@@ -0,0 +1,30 @@
+// External dependencies
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// Internal dependencies
+import UsersList from '@/modules/users/client/components/UsersList';
+import NoContent from '@/modules/core/client/components/NoContent';
+
+export default function UsersResults({ users }) {
+  if (!users || users.length === 0) {
+    return (
+      <NoContent icon="users" message="No members found by this name." />
+    );
+  }
+
+  return (
+    <div className="row">
+      <div className="col-xs-12">
+        <h4 className="text-muted">
+          {users.length === 1 ? 'One member found.' : `${users.length} members found.`}
```

Comment:
We could add translations here, and use https://www.i18next.com/translation-function/plurals.

Reply:
Updated in https://github.com/Trustroots/trustroots/pull/1162/commits/a4cecb07cb9711984bc3ef3cc2ee0c5821336add